conf.yaml

```diff
@@ -5,6 +5,7 @@ meta:
 repository:
     owner: galaxyproject
     name: galaxy
+    next_milestone: "16.04"
```

Comment:
yep. It's pretty great. I would love to have it be a push button thing in jenkins maybe with a manual review step, so we really can have everyone participating and getting releases out the door faster and more regularly.